client/src/app/containers/Game/index.jsx

```diff
@@ -0,0 +1,178 @@
+        let playerColors = activeGame['player_colors'];
+        let teams = {};
+        for (const [playerName, color] of Object.entries(playerColors)) {
+          if (teams[color]) {
+            teams[color] += `, ${playerName}`;
+          } else {
+            teams[color] = `${playerName}`;
+          }
+        }
+        for (const [color, curscore] of Object.entries(val)) {
+          let team = ` (${teams[color]})`;
+          let scoreStr = `${color}${team}: ${curscore}`;
+          if (lagScores && lagScores[color]) {
+            scoreStr += ` (from ${lagScores[color]})`;
+          }
+          if (color === myColor) {
+            scoreChildren.unshift(
+              <strong key='myTeamScore'>{scoreStr}</strong>,
+              <br key={`br ${color}`}></br>,
+            );
+          } else {
+            scoreChildren.push(scoreStr, <br key={`br ${color}`}></br>);
+          }
+        }
+        break;
+      case 'current_dealer':
+        dealerDesc += val;
+        break;
+      case 'phase':
+        phaseDesc += val;
+        break;
+      case 'cut_card':
+        if (
+          activeGame['phase'] === 'Deal' ||
+          activeGame['phase'] === 'BuildCrib'
+        ) {
+          break;
+        }
+        cutCardDiv = (
+          <div key={'cutCardDiv'}>Cut Card: {jsonCardToCard(val)}</div>
+        );
+        break;
+      case 'hands':
+        for (const [playerID, hand] of Object.entries(val)) {
+          if (!hand) {
+            continue;
+          } else if (playerID === currentUser.id) {
+            myHandDiv = (
+              <div key={'myHandDiv'}>My Hand: {hand.map(jsonCardToCard)}</div>
+            );
+          } else {
+            oppHandDivs.push(
+              <div key={`oppHand ${playerID}`}>
+                {playerNamesByID[playerID]}'s Hand:{' '}
+                {hand.length > 0 ? hand.map(jsonCardToCard) : 'empty/unknown'}
+              </div>,
+            );
+          }
+        }
+
+        break;
+      case 'crib':
+        cribDiv = (
+          <div key={'cribDiv'}>Crib: {val.map(jsonCardToCard).join(', ')}</div>
+        );
+
+        break;
+    }
+  }
+
+  return (
+    <div>
+      <div key={'gameDescDiv'}>{gameDesc}</div>
+      <div key={'scoresDiv'}>
+        <h2>Scores:</h2>
+        {scoreChildren}
+      </div>
+      <div key={'dealerDiv'}>{dealerDesc}</div>
+      <div key={'phaseDiv'}>{phaseDesc}</div>
+      {cutCardDiv}
+      {myHandDiv}
+      {oppHandDivs.length > 0 ? (
+        <div key={'oppHandsDiv'}>{oppHandDivs}</div>
+      ) : null}
+      {cribDiv}
+      <br></br>
+      This will be a page for the game of a user.
+      <br></br>
+      {gameResp}
+      <br></br>
+      {refreshButton}
+    </div>
+  );
+};
+
+const jsonCardToCard = card => {
+  // ${card.name}
+  // return `${card.value} ${card.suit}`;
+  return (
+    <PlayingCard
+      key={card.name}
+      name={card.name}
+      value={card.value}
+      suit={card.suit}
+    />
```

Comment:
At this point why not just have `PlayingCard` accept a card object instead of individual values?